lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -89,43 +87,30 @@ define(function(require) {
     },
 
 
-    imageUploadInit: function () {
-      new ImageUpload(this.$uploadButton);
-
-      this.$captionField.hide();
-    },
-
-
     /**
      * Add the reportback entries to the DOM.
      * Adds the entire extended gallery if it does not exist already,
      * otherwise just appends the additional entries.
-     *
-     * @param {Integer} inventoryIndex Index to start appending the next Reportback in gallery.
      */
-    appendEntries: function (inventoryIndex) {
-      var start    = inventoryIndex;
-      var end      = start + this.bundleSize;
+    appendEntries: function () {
+      var start    = this.inventoryIndex;
+      var end      = this.inventory.length;
       var $cluster = $();
       var $images  = null;
 
-      if (this.outOfStock(end)) {
-        end = this.itemsTotal;
-      }
-
       for (var i = start; i < end; i++) {
         if (this.inventory[i] !== undefined) {
           $cluster = $cluster.add(this.inventory[i]);
         }
       }
 
-      $images = $cluster.find("img");
+      $images = $cluster.find('img');
 
       $images.unveil(100, function () {
         var $this = $(this);
 
         $this.load(function () {
-          $this.parent(".photo").addClass("-unveiled");
+          $this.parent('.photo').addClass('-unveiled');
```

Comment:
Should we rename this class to `.is-unveiled` since it's a state class? (Sorry, I know this didn't really change.)